1.10/deploying-services/creating-services/health-checks.md

```diff
@@ -1,23 +1,23 @@
 ---
-post_title: Marathon Health Checks
+post_title: Health Checks
 menu_order: 1
 ---
 
 You can define health checks for your DC/OS services. Health checks are defined on a per application basis, and are run against that application's tasks.
 
-Marathon health checks perform periodic checks on the containers distributed across a cluster to make sure they’re up and responding. If a container is down for any reason, Marathon will detect this and stop sending traffic to the container and try to restart it.
+Health checks perform periodic checks on the containers distributed across a cluster to make sure they’re up and responding. If a container is down for any reason, Marathon will detect this and stop sending traffic to the container and try to restart it.
 
-Health checks begin immediately when a task is launched. By default, health checks are run using the Marathon scheduler. Health checks with the Marathon scheduler only support the COMMAND protocol and each app is limited to a single health check.
+Health checks begin immediately when a task is launched. They are locally executed by Mesos on the agent running the corresponding task and thus test reachability from the Mesos executor. Health checks are performed as close to the task as possible, so they are are not affected by networking failures. Health checks delegated to the agents running the tasks, so the number of tasks that can be health checked can scale horizontally with the number of agents in the cluster.
```

Comment:
Reachability is definitely the wrong word, then. I might just delete that part -- I'm trying to explain that the health check is performed from the perspective of the Mesos executor (and not Marathon), but if DC/OS isn't supporting Marathon-based health checks, it may be irrelevant information.